tests/e2e/e2e_tests_suite_test.go

```diff
@@ -52,13 +54,25 @@ func TestSriovTests(t *testing.T) {
 	RunSpecs(t, "OperatorTests Suite")
 }
 
+var sriovInfos *cluster.EnabledNodes
+var sriovIface *sriovnetworkv1.InterfaceExt
+
 var _ = BeforeSuite(func() {
 	// get global framework variables
 	f := framework.Global
 	// wait for sriov-network-operator to be ready
 	deploy := &appsv1.Deployment{}
 	err := WaitForNamespacedObject(deploy, f.Client, namespace, "sriov-network-operator", RetryInterval, Timeout)
 	Expect(err).NotTo(HaveOccurred())
+	clients := testclient.New("", func(scheme *runtime.Scheme) {
+		sriovnetworkv1.AddToScheme(scheme)
+	})
+	sriovInfos, err = cluster.DiscoverSriov(clients, namespace)
+	Expect(err).ToNot(HaveOccurred())
+	Expect(len(sriovInfos.Nodes)).Should(BeNumerically(">=", 1))
```

Comment:
This is probably not necessary as discoversriov returns an error in case of no sriov enabled nodes.